openreview/conference/templates/areachairWebfield.js

```diff
@@ -837,7 +863,7 @@ var renderTableAndTasks = function(fetchedData) {
   var primaryAssignments = _.filter(fetchedData.blindedNotes, function(note) { return fetchedData.acPapers.indexOf(note.number) > -1; });
   if (paperRankingInvitation) {
     availableOptions = ['No Ranking'];
-    
+
     primaryAssignments.forEach(function(note, index) {
       availableOptions.push((index + 1) + ' of ' + primaryAssignments.length );
     })
```

Comment:
@melisabok At line 878 in new code, we are still using filterfunc. We will need to remove it.

Reply:
thanks for the catch, I removed it. I'm also showing the reviewer rankings.